frontend/src/modules/EditZing/Types/Student.ts

```diff
@@ -9,6 +9,8 @@ export interface Student {
 export interface GroupMembership {
   courseId: string
   groupNumber: number
+  notes: string
+  notesModifyTime: string
```

Comment:
The `notesModifyTime` should be stored as a Date object since that's how other timestamps are represented on the frontend right now. Update the function `responseStudentToStudent` at the bottom of this file to do the conversion.